samples/openthread/cli/README.rst

```diff
@@ -64,6 +64,21 @@ See `Testing diagnostic module`_ section for an example.
 .. note::
     If you disable the :kconfig:option:`CONFIG_OPENTHREAD_NORDIC_LIBRARY_MASTER` feature set, you can enable the diagnostic module with the :kconfig:option:`CONFIG_OPENTHREAD_DIAG` Kconfig option.
 
+Additionally for the ``nrf52840dongle_nrf52840`` build target, diagnostic GPIO commands have been enabled by ``openthread_config`` node in :file:`boards/nrf52840dongle_nrf52840.overlay`.
+This node has been configured for P0.19 pin, which is connected to reset on nRF52840 Dongle.
+By setting it in output mode with low state reset is pulled to GND, which results in device rebooting without skipping the bootloader.
```

Comment:
```suggestion
The pin is set in output mode with a low state  so that reset is pulled to **GND**, which results in the device rebooting without skipping the bootloader.
```
Same as above, should "reset" be "the **RESET** button"?